ci-operator/config/openshift/machine-os-images/openshift-machine-os-images-release-4.13__okd-scos.yaml

```diff
@@ -0,0 +1,46 @@
+base_images:
+  base:
+    name: "4.13"
+    namespace: origin
+    tag: base
+  ocp_4.13_installer:
+    name: scos-4.13
+    namespace: origin
+    tag: installer
+build_root:
+  from_repository: true
+images:
+- build_args:
+  - name: DIRECT_DOWNLOAD
+    value: "true"
+  dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    ocp_4.13_installer:
+      as:
+      - registry.ci.openshift.org/ocp/4.13:installer
```

Comment:
```suggestion
      - registry.ci.openshift.org/origin/scos-4.13:installer
```

Reply:
Doesn't this replace https://github.com/openshift/machine-os-images/blob/main/Dockerfile#L1 and is correct here?